mobile_config/tests/integrations/gateway_db.rs

```diff
@@ -0,0 +1,197 @@
+use chrono::{TimeZone, Timelike, Utc};
+use futures::{pin_mut, StreamExt};
+use helium_crypto::PublicKeyBinary;
+use mobile_config::gateway::db::{Gateway, GatewayType};
+use sqlx::PgPool;
+
+use crate::common;
+
+#[sqlx::test]
+async fn gateway_insert_and_get_by_address(pool: PgPool) -> anyhow::Result<()> {
+    let addr = pk_binary();
+    let now = Utc::now()
+        .with_nanosecond(Utc::now().timestamp_subsec_micros() * 1000)
```

Comment:
Why are we settings nanoseconds?

Reply:
so this happens in test cause one does nano and the other micro so they dont match note this only happens on GA